README.md

```diff
@@ -326,6 +328,80 @@ class PresentationViewController: UIViewController {
 
 You can learn more about `EpoxyPresentations` in its [wiki entry](https://github.com/airbnb/epoxy-ios/wiki/EpoxyPresentations).
 
+## EpoxyLayoutGroups
+
+LayoutGroups are UIKit [Auto Layout](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/index.html) containers inspired by SwiftUI's [`HStack`](https://developer.apple.com/documentation/swiftui/hstack) and [`VStack`](https://developer.apple.com/documentation/swiftui/vstack) that allow you to easily compose UIKit elements into horizontal and vertical groups.
+
+`VGroup` allows you to group components together vertically to create stacked components like this:
+
+| ActionRow |
+| --------- |
+| ![ActionRow screenshot](docs/images/ActionRow.png) |
+
+```swift
+// Set of dataIDs to have consistent and unique IDs
+enum DataID {
+  case title
+  case subtitle
+  case action
+}
+
+// Groups are created declaratively just like Epoxy ItemModels
+let group = VGroup(alignment: .leading, spacing: 8) {
+  Label.groupItem(
+    dataID: DataID.title,
+    content: "Title text",
+    style: .title)
+  Label.groupItem(
+    dataID: DataID.subtitle,
+    content: "Subtitle text",
+    style: .subtitle)
+  Button.groupItem(
+    dataID: DataID.action,
+    content: "Perform action",
+    behaviors: .init { button in
+      print("Button tapped! \(button)")
+    },
+    style: .standard)
+}
+
+// install your group in a view
+group.install(in: view)
+
+// constrain the group like you would a normal subview
+group.constrainToMargins()
+```
```

Comment:
Could we put the code on the left of a table with the screenshot on the right to match the other examples and to limit the length of this README? Or would that not work well here?

Reply:
hmm I can try, it might make the screenshots look very squashed since they are wide and short